theme-settings.php

```diff
@@ -100,26 +100,6 @@ function megatron_form_system_theme_settings_alter(&$form, &$form_state) {
     ),
   );
 
-  $form['clf_navigation_option']['clf_sticky_option'] = array(
```

Comment:
What happened to this hunk of code? I don't see "clf_navoption" for example in the new code hunks

Reply:
This was moved down with the other $form['clf_navigation_option'] options